pyproject.toml

```diff
@@ -26,6 +26,7 @@ python-dateutil = "2.9.0.post0"
 jsonschema = {version = "4.22.0", extras = ["format"]}
 docstring-parser = "^0.16"
 charset-normalizer = "^3.3.2"
+univers = "^30.11.0"
```

Comment:
maybe use a fixed version of univers, just in case the spec is changed? Dependabot and our tests can take care of keeping it up to date and make sure that there are no breaking changes.

Reply:
fixed on the current